lib/measured/rails/base.rb

```diff
@@ -1,7 +1,8 @@
 require "measured/rails/version"
 require "measured"
 
-require "active_support"
+require "active_support/all"
+require "active_record"
```

Comment:
This is explicit. The first comes from measured anyway, and the second we need for extending the activerecord model.